<launch>
	<arg name="namespace" default="" />

	<group ns="$(arg namespace)">
		<!-- Start compressed_image_republisher node -->
		<node name="compressed_image_cropped_republisher" pkg="vibro_tactile_toolbox" type="compressed_image_cropped_republisher.py" output="screen">
			<param name="compressed_image_sub_topic_name" value="side_camera/color/image_raw/compressed" />
			<param name="compressed_image_pub_topic_name" value="side_camera/color/image_cropped/compressed" />
			<param name="x_offset" value="1337" />
			<param name="y_offset" value="1020" />
			<param name="x_size" value="1280" />
			<param name="y_size" value="720" />
			<param name="rotation" value="180" />
		</node>

	</group>
</launch>
